modules/enabling-cluster-capabilites.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="enabling_the_cluster_capabilities_{context}"]
+= Enabling the cluster capabilities
+
+As a cluster administrator, you can enable the capabilities in the cluster by using OpenShift CLI (oc).
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have installed the jq package.
+
+.Procedure
+
+. Add to `additionalEnabledCapabilities` or adjust `baselineCapabilitySet` by running the following command:
++
+[source,terminal]
+----
+$ oc get -o json clusterversion version | jq -c '.status.capabilities.enabledCapabilities' ["openshift-samples"]
```

Comment:
`["openshift-samples"]` is the output which should go to a separate `Example output`

Reply:
done.